backend-diagram-converter/cli/src/main/java/org/camunda/community/migration/converter/cli/ConvertCommand.java

```diff
@@ -6,7 +6,7 @@
 import picocli.CommandLine.Command;
 
 @Command(
-    name = "camunda-convert",
+    name = "java -Dfile.encoding=UTF-8 -jar backend-diagram-converter-cli.jar",
```

Comment:
You mentioned there is another field in the command annotation called description.
We should move this hint there and revert the name change

Reply:
Done.

I've also checked all other options on the help texts and added some of them. It looks much cleaner now.

Reply:
@jonathanlukas, what do you think?

Reply:
Looks good to me!